src/api/app/services/worker_status_service/jobs_statistics_fetcher.rb

```diff
@@ -0,0 +1,33 @@
+module WorkerStatusService
+  class JobsStatisticsFetcher
```

Comment:
:thought_balloon: I think this belongs into the backend library. It's getting data from the backend and molding it into a consumable form.

Reply:
we could do it later, lets experiment with this code without change the backend lib